feature/interface/ip/ipv6_slaac_link_local/otg_tests/ipv6_slaac_link_local_test/ipv6_slaac_link_local_test.go

```diff
@@ -20,17 +22,47 @@ var (
 	reIPv6BySLAAC   = regexp.MustCompile(ipv6BySLAAC)
 )
 
+var (
+	plenIPv4 = uint8(30)
+	plenIPv6 = uint8(126)
+
+	dutPort1 = &attrs.Attributes{
+		Name:    "port1",
+		IPv4:    "192.0.2.1",
+		IPv4Len: plenIPv4,
+		IPv6:    "2001:0db8::192:0:2:1",
+		IPv6Len: plenIPv6,
+	}
+	atePort1 = &attrs.Attributes{
+		Name:    "port1",
+		MAC:     "02:00:01:01:01:01",
+		IPv4:    "192.0.2.2",
+		IPv4Len: plenIPv4,
+		IPv6:    "2001:0db8::192:0:2:2",
+		IPv6Len: plenIPv6,
+	}
+)
+
 func TestMain(m *testing.M) {
 	fptest.RunTests(m)
 }
 
+func configureATEInterface(t *testing.T, ate *ondatra.ATEDevice, p2 *ondatra.Port, top gosnappi.Config) {
+	t.Helper()
+	atePort1.AddToOTG(top, p2, dutPort1)
+	otg := ate.OTG()
+	otg.PushConfig(t, top)
+	otg.StartProtocols(t)
+}
+
 func configureDUTLinkLocalInterface(t *testing.T, dut *ondatra.DUTDevice, p *ondatra.Port) {
 	t.Helper()
 
 	intf := &oc.Interface{Name: ygot.String(p.Name())}
```

Comment:
can you please also change this to
`	intf := dutPort1.NewOCInterface(p.Name(), dut)`

(otherwise the interface is configured without `type` attribute)

Reply:
add `intf.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd`

Reply:
yep, that should work as well